docs/api_requests/logger.md

```diff
@@ -0,0 +1,47 @@
+# Logger API Request
+The Logger can be configured by sending a `PUT` API Request to the `/logger`
+path.
+
+Details about the required fields can be found in the
+[swagger definition](../../api_server/swagger/firecracker.yaml).
+
+
+
+## LogDirtyPages Option
+When the `LogDirtyPages` option is specified in the `options` field, every 60
+seconds a metric with the guest dirty pages count is emitted.
+The dirty_pages number represents the number of pages in the guest memory
```

Comment:
It would look nicer to add some kind of emphasis to the "dirty_pages" mention on this line.